packages/oui-dual-list/src/dual-list.controller.js

```diff
@@ -0,0 +1,247 @@
+export default class {
+    constructor ($q, $element, ouiDualListProvider) {
+        "ngInject";
+
+        this.$q = $q;
+        this.$element = $element;
+        this.dualListProvider = ouiDualListProvider;
+    }
+
+    $onInit () {
+        this.sourceListLabel = this.sourceListLabel || this.dualListProvider.translations.sourceListLabel;
```

Comment:
We should use `addDefaultParameter()`.
We have some component utils for this use cases, in `component-utils.js`. :)

You'll need $attrs in your controller for this